molgenis/client.py

```diff
@@ -64,18 +37,12 @@ def __init__(self, url: str = "http://localhost:8080/", token: str = None):
         Examples:
         >>> session = Session('http://localhost:8080/')
         """
-        self._set_urls(url)
+        self._api_url = utils.set_urls(url)[0]
+        self._root_url = utils.set_urls(url)[1]
```

Comment:
You're calculating the urls twice here. You need to store the result of `set_urls` in a temporary variable, or you can unpack the returned tuple like this:

```python
self._api_url, self._root_url = utils.set_urls(url)
```

Reply:
Won't fix because the current way of defining these urls is restored.